akka-http-core/src/main/mima-filters/10.0.10.backwards.excludes

```diff
@@ -0,0 +1,7 @@
+# Changes to internal API
+ProblemFilters.exclude[MissingClassProblem]("akka.http.impl.model.parser.HeaderParser$RuleNotFoundException$")
```

Comment:
Can we `ProblemFilters.exclude[Problem]("akka.http.impl.*")`?

Reply:
I guess we can. It's somewhat interesting to see what has changed in the API but it's mostly distracting so I'll try.